apps/web/src/app/app.tsx

```diff
@@ -46,11 +77,37 @@ export function App() {
       value={value.children}
       viewport={value.viewport}
       theme={value.theme}
-      onChange={(value) => {
-        localforage.setItem(MAIN_BOARD_CONTENT_KEY, value);
+      //@ts-ignore
+      onChange={(boardData: BoardChangeData) => {
+        // Save theme to localStorage when it changes
+        if (boardData.theme?.themeColorMode) {
+          saveThemeToStorage(boardData.theme.themeColorMode);
+        }
+
+        // Don't save theme to localforage - only save content and viewport
+        const newValue = {
+          children: boardData.children,
+          viewport: boardData.viewport,
+          theme: boardData.theme, // Keep for state but don't persist
+        };
+        setValue(newValue);
+
+        // Only save children and viewport to localforage, not theme
```

Comment:
I am curious that why not save theme to localforage and save it to localStorage.

I think is ok that save viewport, children, and theme to one place.